src/rt.rs

```diff
@@ -65,6 +65,12 @@ pub fn is_initialized_main_thread() -> bool {
 }
 
 /// Informs this crate that GTK has been initialized and the current thread is the main one.
+///
+/// # Safety
+///
+/// You must only call this if you have initialised GTK yourself, on the thread
+/// with which you are calling this function, and that this thread is the main thread
```

Comment:
The reading is a bit complicated here...

Reply:
I wasn't sure if this even ought to be public -- I can expand this into bullets instead of a compound sentence if it does need to remain public, otherwise we could unpub it.

Reply:
Should be public, yes

Reply:
I've pushed replacement docs for this.